docs/src/app/components/pages/components/dialog.jsx

```diff
@@ -72,6 +76,12 @@ export default class DialogPage extends React.Component {
             desc: 'Set to true to have the dialog automatically open on mount.',
```

Comment:
I have never seen deprecation warnings too. We used to just get rid of the property and add it to the breaking changes list when releasing. @shaurya947 What do you want to de here?